liquidator/liquidator.js

```diff
@@ -118,70 +112,82 @@ class Liquidator {
     // expired and disputed liquidations.
     const expiredLiquidations = this.empClient.getExpiredLiquidations();
     const disputedLiquidations = this.empClient.getDisputedLiquidations();
-    const potentialWithdrawableLiquidations = expiredLiquidations.concat(disputedLiquidations);
+    const potentialWithdrawableLiquidations = expiredLiquidations
+      .concat(disputedLiquidations)
+      .filter(liquidation => liquidation.liquidation === this.account);
 
-    if (potentialWithdrawableLiquidations.length > 0) {
-      Logger.info({
+    if (potentialWithdrawableLiquidations.length === 0) {
+      Logger.debug({
+        at: "liquidator",
+        message: "No withdrawable liquidations"
+      });
+      return;
+    }
+
+    Logger.debug({
+      at: "liquidator",
+      message: "potential withdrawable liquidations detected!💰",
```

Comment:
debug should not have an emoji
```suggestion
      message: "potential withdrawable liquidations detected",
```